addons/udes_stock/tests/test_stock_move.py

```diff
@@ -107,3 +124,65 @@ def test04_unreserve_initial_demand(self):
         new_move_lines = MoveLine.search([("product_id", "=", self.fig.id)])
         self.assertEqual(move_lines, new_move_lines)
         self.assertIn(pack2_ml, new_move_lines)
+
+    def test05_pepare_and_create_single_move_line(self):
+        """ Prepare and create a single move line and check values are correct """
```

Comment:
Usually, move lines are created when stock is reserved. After setup there are only 2 units left for bananas, so probably change product_uom_qty to 2 instead of 5 to make it more realistic.

Also the state of the current pick is "assigned" and adding new moves in state draft without confirming/assigning them does not make much sense.

Reply:
Have updated `product_uom_qty` to 2. Picking is now also confirmed with the stock assigned at the end of the test.